app/models/place.rb

```diff
@@ -8,43 +8,19 @@ def validate(record)
   end
 end
 
-class Place
-  include Mongoid::Document
-
+class Place < ApplicationRecord
   # Match documents with either no geocode error or a null value. Changed so
   # that anything without a location (or with a null location) is either
   # matched by `needs_geocoding` or `with_geocoding_errors`.
   scope :needs_geocoding, -> { where(location: nil, geocode_error: nil) }
 
   # We use "not null" here instead of "exists", because it works with the index
-  scope :with_geocoding_errors, -> { where(:geocode_error.ne => nil) }
+  scope :with_geocoding_errors, -> { where.not(geocode_error: nil) }
```

Comment:
This comment points at a workaround to get mongo to use an index that maybe isn't needed for postgres...

Is this still the cleanest way of expressing "where there are geocode errors"?

Is the comment still accurate? If not, we should remove the comment.

Reply:
I think it's still the cleanest way to express it (AFAIK this is the idiomatic way to say "get me all the records where there's *something* in that field), but I think it's so idiomatic that the comment isn't necessary (it might have been for people not familiar with mongo, but this is the standard activerecord way of doing it now, so the comment is superflous). I'll remove.

Reply:
Fixed